packages/fhir-router/src/graphql.ts

```diff
@@ -151,7 +151,8 @@ export async function graphqlHandler(req: FhirRequest, repo: FhirRepository): Pr
   }
 
   const introspection = isIntrospectionQuery(query);
-  if (introspection) {
+  const introspectionEnabled = !!req.options?.introspectionEnabled;
+  if (introspection && !introspectionEnabled) {
```

Comment:
It'd probably be cleaner to just inline the check to avoid triple negation

`if (introspection && !req.options?.introspectionEnabled) {`